pisa/stages/discr_sys/README.md

```diff
@@ -42,39 +42,73 @@ This will generate N different `.json` for N systematics.
 All the info from the fit, including the fit function itself is stored in that file.
 Plotting is also available via `-p/--plot' and is HIGHLY recomended to inspect the fit results.
 
-### Ultrasurfaces
-
-This is the novel treatment of detector systematics via likelihood-free inference. It assigns gradients to _every event_ to allow event-by-event re-weighting as a function of detector uncertainties in a way that is fully decoupled from flux and oscillation effects.
 
-Once ready, the results are stored in a `.feather` file containing all events of the nominal MC set and their associated gradients.
+### Ultrasurfaces
 
-### Preparation
+Treatment of detector systematics via likelihood-free inference. Polynomial coefficients, assigned to every event allow continuoius re-weighting as a function of detector uncertainties in a way that is fully decoupled from flux and oscillation effects. The results are stored in a feather file containing all events of the nominal MC set and their associated polynomial coefficients.
 
-The scripts producing the gradients are located in `$FRIDGE_DIR/analysis/oscnext_ultrasurfaces`. To produce the gradient feather file, we first need to convert PISA HDF5 files to `.feather` using the `pisa_to_feather.py` script. We need to pass the input file, the output file, and a flag setting the sample (variable names) to be used (either `--verification-sample`, `--flercnn-sample`, `--flercnn-hnl-sample`, `--upgrade-sample`, or no additional flag for the Retro sample).
+To use this in a PISA analysis pipeline, you will need to set up an ultrasurface config file looking like this:
```

Comment:
* no need to mention "PISA"
* doesn't need to be a separate config file, perhaps just write "set up a `discr_sys.ultrasurfaces` stage" (this documentation isn't meant to discuss best practices on how to set up a pipeline config or how to use the import syntax)

Reply:
Removed this part as per your general comment.